Modules/Options/GeneralTab/QuestieOptionsGeneral.lua

```diff
@@ -245,6 +247,42 @@ function QuestieOptions.tabs.general:Initialize()
                             QuestieQuest:SmoothReset()
                         end,
                     },
+                    usePfQuestIcons = {
```

Comment:
This toggle should ideally be replaced with a drop-down selection for each icon type, but it is good enough for now and can be used as a basis for a "style" selection frame later.